metadata-ingestion/tests/integration/dremio/dremio_to_file.yml

```diff
@@ -0,0 +1,27 @@
+source:
+  type: dremio
+  config:
+    # Coordinates
+    hostname: localhost
+    port: 9047
+    tls: false
+
+    # Credentials
+    authentication_method: password
+    username: admin
+    password: "2310Admin1234!@"
+
+    include_query_lineage: false
+
+    source_mappings:
+      - platform: s3
+        source_name: samples
```

Comment:
Can we add include platform_instance of source here ? Would improve on test coverage of platform_instance for lineage cases.

Reply:
yes,